src/lib.rs

```diff
@@ -458,8 +458,8 @@ pub(crate) use castable;
 /// This can be used regardless of the [`Castable::Ownership`] as we can make const pointers for
 /// `Box`, `Arc` and ref types.
 pub(crate) fn cast_const_ptr<C>(ptr: *const C) -> *const C::RustType
-where
-    C: Castable,
+    where
```

Comment:
This commit is also reformatting these where clauses. Intended?

Reply:
Actually this reformatting gets undone by the actual formatting commit later, so maybe this could be just dropped?

Reply:
I think this was a mistake along the way. I'll tidy :+1:

Reply:
I force pushed a more careful update that avoids any formatting changes until the very end when the project-wide reformatting is applied.